models/git/branch.go

```diff
@@ -101,9 +101,9 @@ func (err ErrBranchesEqual) Unwrap() error {
 type Branch struct {
 	ID            int64
 	RepoID        int64  `xorm:"UNIQUE(s)"`
-	Name          string `xorm:"UNIQUE(s) NOT NULL"`
+	Name          string `xorm:"UNIQUE(s) NOT NULL"` // git's ref-name is case-sensitive internally, however, in database, it's case-insensitive at the moment
```

Comment:
It's not true. For mysql/mssql the default collations are case-insensitive, for sqlite/postgres/tidb, the default collations are case-sensitive. I will send a PR to fix that.

Reply:
yup, "database related": mssql, mysql , by default.

Reply:
```suggestion
	Name          string `xorm:"UNIQUE(s) NOT NULL"` // git's ref-name is case-sensitive internally, however, in some databases (mssql, mysql, by default), it's case-insensitive at the moment
```